components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -160,199 +205,137 @@ + (NSDictionary *)catalogMetadata {
-  bannerView.frame = CGRectMake(0.0f, 0.0f, bannerViewSize.width, bannerViewSize.height);
+  [bannerView sizeToFit];
 
   // Adjust bannerViewContainer's frame
   CGFloat topAreaInset = 0.0f;
   if (@available(iOS 11.0, *)) {
     topAreaInset = self.view.safeAreaInsets.top;
   }
-  self.bannerViewContainer.frame = CGRectMake(
-      0.0f, topAreaInset, [UIScreen mainScreen].bounds.size.width, bannerViewSize.height);
-  bannerView.center = CGPointMake(self.bannerViewContainer.frame.size.width / 2,
-                                  self.bannerViewContainer.frame.size.height / 2);
+
+  bannerView.frame = CGRectMake(0.0f, topAreaInset, CGRectGetWidth(bannerView.frame),
+                                CGRectGetHeight(bannerView.frame));
+}
+
+- (void)showSingleLineStyleBannerWithIcon {
+  [self showSingleLineStyleBanner];
+  self.bannerView.icon = [UIImage imageNamed:@"Email"];
 }
 
 - (void)showMultiLineAlignedButtonStyleBanner {
-  if (self.bannerViewContainer) {
-    [self.bannerViewContainer removeFromSuperview];
+  if (self.bannerView) {
+    [self.bannerView removeFromSuperview];
   }
-  // Set up banner view container
-  UIView *bannerViewContainer = [[UIView alloc] initWithFrame:CGRectZero];
-  [self.view addSubview:bannerViewContainer];
-  bannerViewContainer.backgroundColor = [UIColor whiteColor];
-  self.bannerViewContainer = bannerViewContainer;
 
   MDCBannerView *bannerView = [[MDCBannerView alloc] init];
+  bannerView.preferredContentWidth = exampleBannerContentWidth;
   bannerView.text = exampleLongText;
-  bannerView.image = [UIImage imageNamed:@"Email"];
-  bannerView.numberOfButtons = 2;
+  bannerView.backgroundColor = self.colorScheme.surfaceColor;
+  [self.view addSubview:bannerView];
+  self.bannerView = bannerView;
 
-  UIFont *buttonFont = [MDCTypography body2Font];
-  MDCButton *dismissButton = bannerView.buttons[0];
+  MDCButton *dismissButton = [[MDCButton alloc] initWithFrame:CGRectZero];
   [dismissButton setTitle:@"DISMISS" forState:UIControlStateNormal];
-  [dismissButton setTitleFont:buttonFont forState:UIControlStateNormal];
-  [dismissButton setTitleFont:buttonFont forState:UIControlStateHighlighted];
   [dismissButton setTitleColor:self.colorScheme.primaryColor forState:UIControlStateNormal];
   dismissButton.backgroundColor = self.colorScheme.surfaceColor;
   [dismissButton sizeToFit];
   [dismissButton addTarget:self
                     action:@selector(dismissBanner)
           forControlEvents:UIControlEventTouchUpInside];
-  MDCButton *changeTextButton = bannerView.buttons[1];
-  [changeTextButton setTitle:@"CHANGE TEXT" forState:UIControlStateNormal];
-  [changeTextButton setTitleFont:buttonFont forState:UIControlStateNormal];
-  [changeTextButton setTitleFont:buttonFont forState:UIControlStateHighlighted];
+  MDCButton *changeTextButton = [[MDCButton alloc] initWithFrame:CGRectZero];
+  [changeTextButton setTitle:@"LONG DISMISS" forState:UIControlStateNormal];
```

Comment:
Tip: The title should be sentence case and `.uppercaseTitle` should be set to `YES` on the button.  Here and throughout.

Reply:
Reasonable. Addressed in 9418df0802d910e1d91d9c79d32a080c6a3fbf75.